### ISSUE TITLE
Can't post links that include ##, claims that it's not valid.

### ISSUE THREAD

**Issue by dylansl** [voat/voat#418]
Tried to submit "https://www.gog.com/games##sort=bestselling&price=free&page=1" and it triggered the red "Please enter a valid http, https, or ftp URL." response. Removing the 2nd pound sign allowed it to be submitted, and entering that changed it to "https://www.gog.com/games#/sort=bestselling&price=free&page=1#sort=bestselling&price=free&page=1" which still isn't allowed.

**Comment by voat** [comment]
We are using [UrlAttribute Class](https://msdn.microsoft.com/en-us/library/system.componentmodel.dataannotations.urlattribute%28v=vs.110%29.aspx) for URL validation.

Gog.com seems to be using their own "standard" for URIs which triggers the validation error. The # characters are allowed by the validator if they are placed correctly in an URI. What Gog.com is doing with the sample link you provided seems to go against the guidelines set out in [RFC3986](https://tools.ietf.org/html/rfc3986) as they are placing the # character in the Path part of the URI rather than in the Fragment part of the URI.

In short, this is not a voat issue. The sample URI you're referring to is labeled as "Abnormal" in the RFC3986 :)
